crates/ruff_linter/src/rules/pydocstyle/rules/sections.rs

```diff
@@ -1834,6 +1838,16 @@ fn missing_args(checker: &Checker, docstring: &Docstring, docstrings_args: &FxHa
     }
 }
 
+fn should_ignore_kwarg_check(checker: &Checker, parameter: &Parameter) -> bool {
+    if let Some(annotation) = &parameter.annotation {
+        checker
+            .semantic()
+            .match_typing_expr(map_subscript(annotation), "Unpack")
+    } else {
+        false
+    }
```

Comment:
Small nit here:

```suggestion
    parameter.annotation.is_some_and(|annotation|
        checker
            .semantic()
            .match_typing_expr(map_subscript(annotation), "Unpack")
    ) 
```

We sometimes make functions like this only take the `&SemanticModel` from `checker.semantic()` instead of a whole `&Checker` too, but I don't feel too strongly about that.

Reply:
Addressed in https://github.com/astral-sh/ruff/pull/21816/commits/1491d7674cd479cc986bfe363d337af3f4336ca0